src/apps/popularFeatured.js

```diff
@@ -57,6 +57,7 @@ export const getData = async (
         .query(popularFeaturedAppsQuery, [
             username,
             limit,
+            config.favoritesGroupIndex,
```

Comment:
heh, I guess this must have just not shown anything as a favorite before, unless the limit for some reason matched the group index